### ISSUE TITLE
All read commands sent with CommandFlags.PreferSlave, but slave is ignored when looking status

### ISSUE THREAD

**Issue by MosheL** [StackExchange/StackExchange.Redis#286]
Hi!

I am using StackExchange.Redis with master/slave installation.

I am reading with CommandFlags.PreferSlave, but when I am looking inside i see no use on the slave. I want to use the slave for reads! the slave is connected but ignored. why?

here is the full `multiplexer.GetStatus()` result:

    redis1:6379: Standalone v3.0.3, master; 16 databases; keep-alive: 00:01:00; int: ConnectedEstablished; sub: ConnectedEstablished, 1 active 
    redis1:6379: int ops=163768, qu=0, qs=0, qc=0, wr=0, sync=163766, async=2, socks=1; sub ops=47, qu=0, qs=0, qc=0, wr=0, subs=1, sync=47, socks=1 
    Circular op-count snapshot; int: 162882+163+126+54+104+128+86+39+54+76+56=163768 (88.60 ops/s; spans 10s); sub: 47 (0.00 ops/s; spans 10s) 
    redis2:6379: Standalone v3.0.3, slave; 16 databases; keep-alive: 00:01:00; int: ConnectedEstablished; sub: ConnectedEstablished, 1 active 
    redis2:6379: int ops=56, qu=0, qs=0, qc=0, wr=0, sync=54, async=2, socks=1; sub ops=47, qu=0, qs=0, qc=0, wr=0, subs=1, sync=47, socks=1 
    Circular op-count snapshot; int: 56 (0.00 ops/s; spans 10s); sub: 47 (0.00 ops/s; spans 10s) 
    Sync timeouts: 0; fire and forget: 0; last heartbeat: 0s ago 

I am using `CommandFlags.FireAndForget  + CommandFlags.DemandMaster` for write commands, and  `CommandFlags.PreferSlave` for read commands.

thanks !

**Comment by NickCraver** [comment]
Can you provide some sample code here? It could be the exact commands that will shed some light.

**Comment by MosheL** [comment]
```` VB
	Public Const ReadFlags As ClientFlags = CommandFlags.PreferSlave
    ....

    db.StringGet(Name, ReadFlags) 
````

It works. but when I am looking `GetStatus()` and network traffic I see traffic only to remote master and the slave is not used.

**Comment by MosheL** [comment]
fixed after upgrade `Redis` to `3.2`.
